examples/context/theme-detailed-app.js

```diff
@@ -11,18 +11,20 @@ function Toolbar(props) {
 }
 
 class App extends React.Component {
-  state = {
-    theme: themes.light,
-  };
-
-  toggleTheme = () => {
-    this.setState(state => ({
-      theme:
-        state.theme === themes.dark
-          ? themes.light
-          : themes.dark,
-    }));
-  };
+  constructor(props) {
+    this.state = {
+      theme: themes.light,
+    };
+  
+    this.toggleTheme = () => {
```

Comment:
Note: this is still using property initializer syntax

Reply:
Isn't it the same as .bind in the constructor?

Reply:
Yes but it's not in any stable standard. It relies on the "public fields" proposal which is in stage 3.

Reply:
Ooh never mind I completely missed that you did this *inside* the constructor. I guess it's not very idiomatic (people tend to use `bind` instead so that the method is defined together with other class methods), but it's fine.

Reply:
Will change if you think it's confusing. 

https://twitter.com/alexmkrolick/status/977277592087666688